README.md

```diff
@@ -23,37 +23,38 @@ coreos-installer.
 
 # On Fedora CoreOS or RHEL CoreOS
 
-coreos-installer is included in Fedora CoreOS and RHEL CoreOS.  Just run
+coreos-installer is included in Fedora CoreOS.  Just run
 `coreos-installer` from the command line.  Fedora CoreOS provides
 [live CD and network boot images](https://getfedora.org/coreos/download/)
 you can run from RAM; you can use these to run coreos-installer to install
 Fedora CoreOS or RHEL CoreOS to disk.
 
-# On other Fedora editions
+RHEL CoreOS also includes this, but note that versions &lt;= 4.4 contain
+the `legacy` branch of this software.
 
-On other Fedora editions, you can install coreos-installer with DNF in the
-usual way:
+# Run from a container
+
+You can run coreos-installer from a container.  You'll need to bind-mount
+`/dev` and `/run/udev`, as well as a data directory if you want to access
+files in the host.  For example:
 
 ```sh
-sudo dnf install coreos-installer
+sudo podman run --pull=always --privileged --rm \
+    -v /dev:/dev -v /run/udev:/run/udev -v .:/data \
+    quay.io/coreos/coreos-installer:release \
+    install /dev/vdb -s testing -i /data/config.ign
 ```
 
-# Build and install from source tree
+# Via a Fedora RPM
 
-To build from the source tree:
+`coreos-installer` is packaged in Fedora:
 
 ```sh
-make
+sudo dnf install coreos-installer
 ```
 
-To install the binary and systemd units to a target rootfs
-(e.g. under a
-[coreos-assembler](https://github.com/coreos/coreos-assembler)
-workdir):
-
-```sh
-make install DESTDIR=/my/dest/dir
-```
+(Note you can do this inside a `--privileged` container, not necessarily
+ on a host system's root filesystem)
```

Comment:
Could we be clearer here?  For people used to traditional Fedora Workstation/Server, this statement might be confusing.

Reply:
I tweaked this a bit but ultimately I think we want to capture more of this at https://github.com/containers/toolbox  (once it's not completely broken in the case of running privileged)